bot/exts/moderation/clean.py

```diff
@@ -87,17 +85,16 @@ def mod_log(self) -> ModLog:
 
     @staticmethod
     def _validate_input(
-            traverse: int,
             channels: Optional[CleanChannels],
             bots_only: bool,
             users: Optional[list[User]],
             first_limit: Optional[CleanLimit],
             second_limit: Optional[CleanLimit],
     ) -> None:
         """Raise errors if an argument value or a combination of values is invalid."""
-        # Is this an acceptable amount of messages to traverse?
-        if traverse > CleanMessages.message_limit:
-            raise BadArgument(f"Cannot traverse more than {CleanMessages.message_limit} messages.")
+        if first_limit is None:
```

Comment:
We could remove the check on line 111 as `first_limit` is now a required argument, so it wouldn't be `None` (after this check).

Reply:
True